neo/rawio/intanrawio.py

```diff
@@ -1065,6 +1120,12 @@ def read_rhd(filename, file_format: str):
 # This section provides all possible headerless binary files in both the rhs and rhd
 # formats.
 
+digital_stream_names = ["USB board digital input channel", "USB board digital output channel"]
+name_to_field_name_digital = {
```

Comment:
Technically we could just check `stream_name in name_to_field_name_digital` and get rid of line 1123, but you are going for clarity over reduced lines I think.

I would still prefer to think if we have a better naming than `name_to_field_name_digital`, but we are using so many terms maybe this is the best.

Reply:
The name is bad, I agree. Plus, this is bad practice because there is an implicit coupling between the values here and the lines here:

https://github.com/h-mayorquin/python-neo/blob/2af13478edbe92e55a8d308ef7fc8a29220ada40/neo/rawio/intanrawio.py#L799-L801

In other words, a relationship that is implicit. That should not be there.

Reply:
OK, I decided that for this cases is better to use the stream_name as the field name (as all the data is packed in one field). This avoids us having to define an artificial channel name (as we were doing) and sidesteps the need to define such mapping between stream and the artificial channel name. It also works nicely with the case of one-file-per-signal as the signal is the stream.